exporter/exporterhelper/common.go

```diff
@@ -171,7 +190,7 @@ type baseExporter struct {
 	qrSender *queuedRetrySender
 }
 
-func newBaseExporter(cfg config.Exporter, logger *zap.Logger, bs *baseSettings) *baseExporter {
+func newBaseExporter(cfg config.Exporter, logger *zap.Logger, bs *baseSettings, signalName string, reqUnnmarshaler requestUnmarshaler) *baseExporter {
```

Comment:
What is `signalName` used for? What are the allowed values? Are these only `logs`, `metrics` and `traces`?

If so I suggest to make this a separate type like so


```go
type Signal string
```

And define a set of allowed values as `const`s

```go
const (
    SignalLogs = Signal("logs")
...
)
```